src/wix/WixToolset.Core/Linker.cs

```diff
@@ -119,8 +122,8 @@ public Intermediate Link(ILinkContext context)
                     }
                 }
 
-                // Add the missing standard action and directory symbols.
-                this.LoadStandardSymbols(find.SymbolsByName);
+                ////// Add the missing standard action and directory symbols.
```

Comment:
clean up (unless it's one cleaned up in a later commit)

Reply:
Yep. Cleaned up in subsequent commit.